requirements.txt

```diff
@@ -7,6 +7,8 @@ scipy==1.13.1; platform_python_implementation != "PyPy" and python_version <= '3
 scipy==1.14.1; platform_python_implementation != "PyPy" and python_version >= '3.10'
 munkres==1.1.4; platform_python_implementation == "PyPy"
 zopfli==0.2.3.post1
+# fs is no longer required by the 'ufo' extra (only used if present) but we keep it in
+# here so that tox continues to install it for the tests
```

Comment:
But we also need a run without fs, right?

Reply:
we do. the CI runs tox py-noextra as well